xcube/core/verify.py

```diff
@@ -81,9 +81,41 @@ def verify_cube(dataset: xr.Dataset) -> List[str]:
     if xy_var_names and time_var_name:
         _check_data_variables(dataset, xy_var_names, time_var_name, report)
 
+    if xy_var_names:
+        _check_coord_equidistance(dataset, xy_var_names[0], xy_var_names[0], report)
+        _check_coord_equidistance(dataset, xy_var_names[1], xy_var_names[1], report)
+
     return report
 
 
+def _check_coord_equidistance(dataset, coord_name, dim_name, report, rtol=None):
+    diff = dataset[coord_name].diff(dim=dim_name)
+    if not _check_equidistance_from_diff(diff, rtol=rtol):
+        report.append(f"Coordinate {coord_name} is not equidistant")
```

Comment:
Always try being consistent! Here be consistent with other report strings in this module. Therefore

```suggestion
        report.append(f"coordinate variable {coord_name!r} is not equidistant")
```